### PR TITLE
provide bogus user agent to avoid '418 I'm a teapot'

### PR DESCRIPTION
**Pull request by TheSnoozer**
Avoid
```
>>> import requests
>>> URL = "https://production.dataviz.cnn.io/index/fearandgreed/graphdata"
>>> r = requests.get(URL)
>>> r.text
"I'm a teapot. You're a bot."
>>> r.status_code
418
```

### CODE REVIEW

File: src/fear_and_greed/cnn.py [hunk 1]
```diff
@@ -26,7 +26,13 @@ class Fetcher:
     """Fetcher gets the HTML contents of CNN's Fear & Greed Index website."""
 
     def __call__(self) -> dict:
-        r = requests.get(URL)
+        headers = {
+            'User-Agent': 'Mozilla',
+        }
+        r = requests.get(URL, headers=headers)
+        if r.status_code != 200:
```
Comment (TheSnoozer) [inline src/fear_and_greed/cnn.py:33]: As suggested in the linked ticket you could also use `r.raise_for_status()`.

**Comment by vterron** [comment]
I can confirm this fix with `curl`:

```
❯ curl https://production.dataviz.cnn.io/index/fearandgreed/graphdata                       
I'm a teapot. You're a bot.
❯ curl --user-agent Mozilla https://production.dataviz.cnn.io/index/fearandgreed/graphdata
{"fear_and_greed": {"score": 20.0977, "rating": "extreme fear" [...]
```

**Comment by vterron** [comment]
Thanks a lot for contributing this!

File: src/fear_and_greed/cnn.py (hunk 1 above)
Comment (vterron) [reply src/fear_and_greed/cnn.py:33]: Yes, I switched to that approach in 25c9895adb654a20c35179524de6d42ddd0a47b4.

Comment (TheSnoozer) [reply src/fear_and_greed/cnn.py:33]: Cool stuff. Thanks for the quick response and the v0.3 release to pypi.

**Pull request merged**